examples/testplans/release/pre.json

```diff
@@ -10,6 +10,9 @@
 	{"name": "Avocado source is sound",
 	 "description": "On your development machine, on a fresh Avocado source to be released, run `$ make check`. Expected result: Make command should say OK."},
 
+	{"name": "Set the readthedocs.org token",
+	 "description": "Export the content of a token (from https://readthedocs.org/accounts/tokens/) as the AVOCADO_READTHEDOCS_TOKEN environment variable."},
+
```

Comment:
@clebergnu, we are trying to reduce the number of items here. So I'm wondering if we couldn't get this step by any other way. For instance , get the badge forcing with a non cache.

Not sure if is possible, just brain storming here to see if we could remove on step from this list.

Reply:
I see one way to do it automatically. I added myself as the assignee of #3456, and I'll try to come up with an RFC by the end of this sprint.

Reply:
> @clebergnu, we are trying to reduce the number of items here. So I'm wondering if we couldn't get this step by any other way. For instance , get the badge forcing with a non cache.
> 

You mean asking users to forcefully reload? Or fetching the badge image and trying to make sense of it (like OCR-ing it) ?

> Not sure if is possible, just brain storming here to see if we could remove on step from this list.

Sure.  TBH, this should be a "new step" only once, because it's trivial and expected to add `AVOCADO_READTHEDOCS_TOKEN` to one's shell initialization.

Reply:
> I see one way to do it automatically. I added myself as the assignee of #3456, and I'll try to come up with an RFC by the end of this sprint.

Could you share some sneak peaks about your plans?  FIY, we used to build the docs as a test, but the troubleshooting and maintenance was not worth it and we dropped that.

Reply:
If we didn't find the way how to automate this the description have to be better. I had to generate the token before the exporting, and also I think you have to be admin of avocado-framework project on `readthedocs.org` to have a valid token. I would use something like.
 ```
Generate and export the content of a token (from https://readthedocs.org/accounts/tokens/) as the AVOCADO_READTHEDOCS_TOKEN environment variable. For this step you have to be admin of avocado-framework project on `readthedocs.org`
```

Reply:
> I like the idea of using GH Actions encrypted secrets. I'm just not sure if we need this extra step on the pre release testplan.

I agree. If everything goes fine, we won´t need this step anymore.

Reply:
> > > @clebergnu, we are trying to reduce the number of items here. So I'm wondering if we couldn't get this step by any other way. For instance , get the badge forcing with a non cache.
> > 
> > 
> > You mean asking users to forcefully reload? Or fetching the badge image and trying to make sense of it (like OCR-ing it) ?
> 
> I was thinking about doing a curl/request with `-H 'Cache-Control: no-cache` or something similar. I'm just trying to reduce the number of steps, because, unless I'm missing something, this conflicts with #3456.
> 

Sure, but doing a "no cache" request on what?  I did try to find something that a machine could parse, and did not require authentication, but I only succeeded in find one or the other (either API or the badge).

> > I'm counting on this PR to be merged! As there is a secrets functionality on GutHub Actions (https://docs.github.com/en/actions/reference/encrypted-secrets), we can set the token at the GitHub project UI and set this test to run weekly.
> 
> I like the idea of using GH Actions encrypted secrets. I'm just not sure if we need this extra step on the pre release testplan.

+1.  A (minor?) issue is that the the readthedocs.org tokens are personal, and not owned by a project.

Reply:
> If we do not find the way how to automate this, I think the description have to be better. When I was trying that, I had to generate the token before the exporting, and also you have to be admin of avocado-framework project on readthedocs.org to have a valid token. I would use something like.
> 
> ```
> Generate and export the content of a token (from https://readthedocs.org/accounts/tokens/) as the AVOCADO_READTHEDOCS_TOKEN environment variable. For this step you have to be admin of avocado-framework project on `readthedocs.org`
> ```

@richtja I force pushed with your suggestion.